message.go

```diff
@@ -168,13 +168,68 @@ type MessageReactions struct {
 
 // ContentWithMentionsReplaced will replace all @<id> mentions with the
 // username of the mention.
-func (m *Message) ContentWithMentionsReplaced() string {
-	if m.Mentions == nil {
-		return m.Content
+func (m *Message) ContentWithMentionsReplaced() (content string) {
+	content = m.Content
+
+	for _, user := range m.Mentions {
+		content = strings.NewReplacer(
+			"<@"+user.ID+">", "@"+user.Username,
```

Comment:
Realized that, but didn't bother to fix it, because for some reason I thought I'd have to make a bunch of workarounds. Didn't realize it was as easy as changing that :stuck_out_tongue: